### PR TITLE
Change upload image permission

### PR DESCRIPTION
**Pull request by henryyang42**
Change upload image permission to subjudge user (by default is django admin QAQ)

### CODE REVIEW

**Comment by nickelcheng** [comment]
+1, @chiahsinhou please test it

**Comment by henryyang42** [comment]
403 issue fixed.

**Comment by henryyang42** [comment]
Change upload auth to subjudge

**Comment by CHIAHSINHOU** [comment]
This PR is ok for me.

File: nthuoj/settings.py [hunk 1]
```diff
@@ -22,9 +22,9 @@
 SECRET_KEY = 'kivl1x)by8$98z6y3b^7texw&+d1arad2qlq-(sn=8g^lw_(+&'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```
Comment (nickelcheng) [inline nthuoj/settings.py]: Did you mean it?

Comment (henryyang42) [inline nthuoj/settings.py]: Yes

**Comment by nickelcheng** [comment]
ok, going to merge.

**Pull request merged**